src/components/layouts/base-layout.tsx

```diff
@@ -36,6 +36,7 @@ const BaseLayout: React.FC<{ isHome?: boolean }> = ({ children }) => {
               <Flex
                 flexDirection={['column', 'column', 'row', 'row']}
                 maxWidth={`${CONTENT_MAX_WIDTH}px`}
+                pl={4}
```

Comment:
we try to avoid using the number values for spacing, see a list of named literals here: https://github.com/blockstack/ui/blob/main/packages/ui-theme/src/literal-types.ts#L13 and values they map to here https://github.com/blockstack/ui/blob/main/packages/ui-theme/src/sizes.ts#L23

```suggestion
                pl="base"
```